src/main/resources/site/parts/searchResult/searchResult.ts

```diff
@@ -146,9 +146,12 @@ export function renderPart(req: Request): React4xpResponse {
       bestBetResult = {
         title: title,
         preface: firstBet.data.linkedContentIngress ? firstBet.data.linkedContentIngress : '',
-        contentType: firstBet.data.linkedContentType ? firstBet.data.linkedContentType : '',
+        contentType: firstBet.data.linkedContentType ? localize({
+          key: `contentType.search.${firstBet.data.linkedContentType}`,
+          locale: language
+        }) : '',
         url: href,
-        mainSubject: firstBet.data.linkedContentSubject ? firstBet.data.linkedContentSubject : '',
+        mainSubject: firstBet.data.linkedContentSubject ? getSubjectForLanguage(firstBet) : '',
```

Comment:
Should firstBet.data.linkedEnglishContentSubject be included in the conditional?

Reply:
Don't think it's necessary. See the function getSubjectForLanguage, it checks if the english field exists. If it does not, it returns the default field, which is checked here.

Reply:
Added a comment in the code.